From 4321cbc2af788827e56e5581f56d7da2876b48f1 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella Netto <adhemerval.zanella@linaro.org>
Date: Wed, 26 Oct 2022 16:04:25 -0300
Subject: [PATCH 078/118] time: Use 64 bit time on tzfile

The tzfile_mtime is already compared to 64 bit time_t stat call.
Reviewed-by: DJ Delorie <dj@redhat.com>

(cherry picked from commit 4e21c2075193e406a92c0d1cb091a7c804fda4d9)
---
 time/tzfile.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/time/tzfile.c b/time/tzfile.c
index dd75848ba95d8f6c72fd9a6d54894804e5419c36..394b098856da391e5b2b63098cb7e1ae5209d3ad 100644
--- a/time/tzfile.c
+++ b/time/tzfile.c
@@ -32,7 +32,7 @@
 int __use_tzfile;
 static dev_t tzfile_dev;
 static ino64_t tzfile_ino;
-static time_t tzfile_mtime;
+static __time64_t tzfile_mtime;
 
 struct ttinfo
   {
-- 
2.41.0

